src/dotnet/Fable.Compiler/Transforms/Fable2Babel.fs

```diff
@@ -1302,7 +1302,7 @@ module Util =
                     |> declareModuleMember info.IsPublic info.EntityName false
             if info.Entity.IsFSharpExceptionDeclaration
             then yield coreValue com ctx "Types" "FSharpException" |> inherits com ctx consIdent
-            elif info.Entity.IsFSharpRecord
+            elif info.Entity.IsFSharpRecord || info.Entity.IsValueType
```

Comment:
I removed this because I noticed value types don't have structural equality by default in .NET F#, do you need it?

Reply:
Hmm, I'm checking the equality checks in the tests below and they eval to false in my computer's FSI. But the tests pass in Appveyor. Is it because I have an old FSI?

Microsoft (R) F# Interactive version 4.1

Reply:
See above. I thought all things have structural equality/comparison by default in F# (especially structs, it's in the name), unless they have declared CustomEquality/CustomComparison. From [here](https://blogs.msdn.microsoft.com/dsyme/2009/11/08/equality-and-comparison-constraints-in-f/):
> Tuple types support "structural" equality. The same applies to lists, options, arrays and user-defined record, union and struct types whose constituent field types permit structural equality, hashing and comparison.